sandpack-react/src/types.ts

```diff
@@ -108,6 +108,7 @@ export interface SandpackOptions {
   showConsole?: boolean;
   closableTabs?: boolean;
   wrapContent?: boolean;
+  resizablePanel?: boolean;
```

Comment:
new prop name, which defaults to `true`

Reply:
should resizable=true be the default behavior?